stan/math/prim/scal/fun/logical_lte.hpp

```diff
@@ -16,7 +15,7 @@ namespace math {
  * @return <code>true</code> iff <code>x1 &lt;= x2</code>
  */
 template <typename T1, typename T2>
-inline int logical_lte(const T1 x1, const T2 x2) {
+inline bool logical_lte(const T1 x1, const T2 x2) {
```

Comment:
Re-reading meyers C++11 book, wouldn't mind these being

```cpp
inline auto logical_lte(const T1 x1, const T2 x2) {
  return static_cast<bool>(x1 <= x2;);
}
```

Reply:
Question with this one, what's the benefit of ```static_cast<bool>``` and an ```auto``` return type over just having a ```bool``` return type? Won't both approaches just return a ```bool```?

Reply:
Yeah actually ignore this comment I think it's unnecessary